edx_event_bus_kafka/__init__.py

```diff
@@ -2,4 +2,34 @@
 Kafka implementation for Open edX event bus.
 """
 
+from openedx_events.event_bus import EventBus
+
+from edx_event_bus_kafka.consumer.event_consumer import consume_indefinitely
+from edx_event_bus_kafka.publishing.event_producer import send_to_event_bus
+
 __version__ = '0.1.0'
+
+
+class EventBusKafka(EventBus):
+    """Kafka implementation of the EventBus API."""
+
+    def __init__(config):
+        ...  # TODO Read config here and create producer and consumer?
+
+    def send(
+            self, *,
+            signal: OpenEdxPublicSignal, topic: str, event_key_field: str, event_data: dict, sync: bool
+    ) -> None:
+        return send_to_event_bus(
+            signal=signal, topic=topic, event_key_field=event_key_field, event_data=event_data, sync=sync
+        )
```

Comment:
...except I think we need to switch to creating the producer just once, and reusing it, since it's stateful (maintains a buffer, does batching etc.) And that producer object should live in this class instance.